modules/map/utils/normalizePoint.ts

```diff
@@ -0,0 +1,91 @@
+import i18n from '@/i18n.config'
+
+import { KindsEnum } from '../constants'
```

Comment:
Please prefer more explicit name - "KindsEnum" - kind of what?
e.g. `MapPointKindEnum` is may be more describing